packages/loader/lib/index.js

```diff
@@ -6,11 +6,10 @@
  */
 
 import {SourceMapGenerator} from 'source-map'
-import {getOptions} from 'loader-utils'
 import {compile} from '@mdx-js/mdx'
 
 /**
- * A Webpack (4+) loader for MDX.
+ * A Webpack (5+) loader for MDX.
```

Comment:
Does this mean webpack 4 is no longer supported?
This may be okay long term, short term this would be problematic for CRA adopters (pending https://github.com/facebook/create-react-app/discussions/11278 going GA)

Reply:
Yes
I’m not sure the RC / xdm works with webpack 4 anyway?

Reply:
I have recently done some stuff locally though. Might be CodeSandbox acting up? 🤷‍♂️

Reply:
Could be?
I spun up a fresh sandbox for these to try to avoid dependency caching issues, but it may still be an issue :thinking:

Reply:
Well if webpack 4 doesn’t seem to work now anyway, then this isn’t a breaking change.
We can see when folks try the RC out / try it out ourselves locally and go from there later?